### PR TITLE
Login changes

### PR DESCRIPTION
**Pull request by talatcanoguz**
I think this is the PR Neal made to the wrong repository. Can you confirm @nealhmakwana?

### CODE REVIEW

**Comment by talatcanoguz** [comment]
I am guessing all changes to ios directory are to implement firestore/firebase?

File: lib/views/screens/signin_view.dart [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import 'dart:ffi';
+
```
Comment (Thomas-Chambers) [inline lib/views/screens/signin_view.dart:2]: unused import zzz

File: lib/views/widgets/calendar.dart [hunk 2]
```diff
@@ -2,23 +2,42 @@ import 'package:flutter/material.dart';
...
 import 'dart:ui'; // Import this for ImageFilter.
 
-class CalendarWidget extends StatelessWidget {
+class CalendarWidget extends StatefulWidget {
+  @override
+  _CalendarWidgetState createState() => _CalendarWidgetState();
+}
+
+class _CalendarWidgetState extends State<CalendarWidget> {
+  // Define random workout days for the sake of this implementation
```
Comment (Thomas-Chambers) [inline lib/views/widgets/calendar.dart:11]: very descriptive comments, I wonder who wrote them!

File: lib/views/widgets/calendar.dart
Comment (Thomas-Chambers) [inline lib/views/widgets/calendar.dart:1]: like 30 things in here have warnings to use const to improve performance. Not an issue unless we start seeing performance issues in which case we should probably fix

File: lib/views/screens/profile_view.dart [hunk 3]
```diff
@@ -1,16 +1,59 @@
...
+      'Date of Birth': userViewModel.user.dateOfBirth,
+      'Gender': userViewModel.user.gender,
+      'Weight': userViewModel.user.weight,
+      'Height': userViewModel.user.height,
+      'Injury Type': userViewModel.user.injuryType,
+      'Date of Injury': userViewModel.user.dateOfInjury,
+      'Past Injuries': userViewModel.user.pastInjuries,
+      'Total Workouts': userViewModel.user.totalWorkouts
+    };
+  }
```
Comment (Thomas-Chambers) [inline lib/views/screens/profile_view.dart:58]: what about for doctors

File: lib/views/widgets/calendar.dart [hunk 4]
```diff
@@ -2,23 +2,42 @@ import 'package:flutter/material.dart';
 import 'package:table_calendar/table_calendar.dart';
 import 'dart:ui'; // Import this for ImageFilter.
```
Comment (Thomas-Chambers) [inline lib/views/widgets/calendar.dart:3]: redundant import, not needed

**Review by Thomas-Chambers — APPROVED** [review]
would prefer if you did not merge before adding the doctor stuff into profile but I won't stop you from creating tech debt

**Comment by Thomas-Chambers** [comment]
> I am guessing all changes to ios directory are to implement firestore/firebase?

yes

File: lib/views/widgets/calendar.dart
Comment (talatcanoguz) [reply lib/views/widgets/calendar.dart:1]: Added consts for now, these might be removed in the future tho.

**Review by Thomas-Chambers — DISMISSED** [review]
lgtm

**Pull request merged**